src/platforms/javascript/guides/vue/features/component-tracking.mdx

```diff
@@ -47,19 +47,25 @@ The default is `false`.
 
 #### `hooks`
 
-Control which lifecycle hooks should be tracked. This is helpful if, for example, you want to know if some components are removed during the initial page load, in which case you can add a `destroy` hook to the default:
+Control which lifecycle hooks should be tracked. This is helpful if, for example, you want to know if some components are removed during the initial page load, in which case you can add a `unmount` hook to the default:
 
 ```javascript
 Sentry.init({
   // ...
   trackComponents: true
-  hooks: ["mount", "update", "destroy"],
+  hooks: ["mount", "update", "unmount"],
 });
 ```
 
-The following hooks are available to track: `['activate', 'create', 'destroy', 'mount', 'update']`
+The following hooks are available to track in Vue 3: `['activate', 'create', 'unmount', 'mount', 'update']`
 
-Note that when specifying `hooks`, we use the simple verb rather than `before` and `-ed` pairs. For example, `destroy` is correct, while `beforeDestroy` and `destroyed` are incorrect.
+Note that when specifying `hooks`, we use the simple verb rather than `before` and `-ed` pairs. For example, `unmount` is correct, while `beforeUnmount` and `unmounted` are incorrect.
+
+<Alert>
+
+In Vue 2, use `destroy` instead of `unmount`. `destroy` does not work in Vue 3, as the names of the lifecycle hooks themselves [have changed](https://v3-migration.vuejs.org/breaking-changes/#other-minor-changes) in Vue 3.
```

Comment:
```suggestion
In Vue 2, use `destroy` instead of `unmount`. `destroy` does not work in Vue 3, as the names of the lifecycle hooks themselves [changed](https://v3-migration.vuejs.org/breaking-changes/#other-minor-changes) in Vue 3.
```